### DISCUSSION TITLE
Make `mi<char>` consistent

### DISCUSSION THREAD

**Discussion by leath-dub** [helix-editor/helix#11803] [Ideas] [answered]
Maybe I am missing something here, but it seems like the `mi<char>` action is inconsistent. If we compare quote delimited block `"<stuff ...>"` with a brace delimited block `(<stuff ...>)`.

If I have my cursor on the first quote and put in the keys `mi"` it does not select anything else. In contrast if my cursor is on the first bracket and put in the keys `mi(` it selects everything inside of the brackets.

I definitely prefer the latter behaviour, it is how vim behaves too, and is useful as it allows you to go `f"mi"` for example, rather than `f"lmi"`.

**Comment by nik-rev** [comment, marked as answer]
this is worked on in https://github.com/helix-editor/helix/pull/12494
